src/System/Taffybar/Widget/Generic/PollingLabel.hs

```diff
@@ -48,9 +51,27 @@ pollingLabelWithVariableDelay
   => IO (T.Text, Maybe T.Text, Double)
   -> m GI.Gtk.Widget
 pollingLabelWithVariableDelay action =
+  pollingLabelWithVariableDelayAndRefresh action False
+
+-- TODO: Customize the delay and message on mouse click
+pollingLabelWithVariableDelayAndRefresh
```

Comment:
Why not wrap this in the other direction?

Reply:
i.e. make it so that the pollingLabelWithVariableDelayAndRefresh wraps  pollingLabelWithVariableDela

Reply:
Uh, both of them are possible options. My choice is because some other modules in this project are also in this style (e.g. `pollingGraphNew` wraps `pollingGraphNewWithTooltip` in `System.Taffybar.Widget.Generic.PollingGraph`). And it is common to write a most general version of a function and wrap others with it in many Haskell packages like GHC `base`. What's more, this can merge the "internal" details into one place rather than scattering them in many functions.